pkg/sentry/kernel/kernel.go

```diff
@@ -1550,14 +1569,24 @@ func (k *Kernel) RecordSocket(sock *fs.File) {
 // vfs.FileDescription, because we do not support weak refs on VFS2 files.
```

Comment:
nit: "because we do not support weak refs on VFS2 files" can be removed. Eventually, VFS2 will completely replace VFS1 and hopefully weak refs will be gone forever :)

Reply:
Cool.  :)